rust/src/measurements/make_private_expr/mod.rs

```diff
@@ -56,21 +65,94 @@ pub trait PrivateExpr<MI: Metric, MO: Measure> {
     ) -> Fallible<Measurement<ExprDomain, Expr, MI, MO>>;
 }
 
-impl<M: UnboundedMetric + OuterMetric> PrivateExpr<PartitionDistance<M>, MaxDivergence<f64>>
-    for Expr
-{
+impl<M: 'static + UnboundedMetric> PrivateExpr<PartitionDistance<M>, MaxDivergence<f64>> for Expr {
     fn make_private(
         self,
         input_domain: ExprDomain,
         input_metric: PartitionDistance<M>,
-        _output_measure: MaxDivergence<f64>,
+        output_measure: MaxDivergence<f64>,
         global_scale: f64,
     ) -> Fallible<Measurement<ExprDomain, Expr, PartitionDistance<M>, MaxDivergence<f64>>> {
         if expr_laplace::match_laplace(&self)?.is_some() {
             return expr_laplace::make_expr_laplace(input_domain, input_metric, self, global_scale);
         }
 
         match self {
+            #[cfg(feature = "contrib")]
+            Expr::KeepName(expr) => {
+                expr_postprocess::make_expr_postprocess(
+                    input_domain,
+                    input_metric,
+                    output_measure,
+                    vec![*expr],
+                    move |exprs| {
+                        let [expr] = <[Expr; 1]>::try_from(exprs)
+                            .expect("Will always have exactly one expression.");
+                        Ok(expr.name().keep())
+                    },
+                    global_scale
+                )
+            }
+            #[cfg(feature = "contrib")]
+            Expr::Alias(expr, name) => {
+                expr_postprocess::make_expr_postprocess(
+                    input_domain,
+                    input_metric,
+                    output_measure,
+                    vec![*expr],
+                    move |exprs| {
+                        let [expr] = <[Expr; 1]>::try_from(exprs)
+                            .expect("Will always have exactly one expression.");
```

Comment:
Slight preference for this error string to be distinct from the one above: Mention "alias" here? I'm imagining someone getting this message, and either trying to figure out what they did wrong, or grepping the code, and maybe landing at the previous section rather than this one.